### DISCUSSION TITLE
Why there is no nest*.ww3 after running ww3.shell?

### DISCUSSION THREAD

**Discussion by abayhaqi** [NOAA-EMC/WW3#1058] [Q&A]
Hi All, 
I am new to WW3. I have run a model. I set the output request of Type 1 (field mean wave parameters) and Type 5 (nesting data) in ww3.shel (please see the attachments). However, why  the output file of nest*.ww3 does not appear after the ww3.shel running is finished?
![ww3shel2](https://github.com/NOAA-EMC/WW3/assets/49505651/7f44b65f-1164-4bc9-9b22-2787ca240a33)
![ww3shel1](https://github.com/NOAA-EMC/WW3/assets/49505651/c15040c8-a258-443b-991d-f991111f62e0)

**Comment by EmilioEchevarria** [comment]
Did you also define your nest(s) in ww3_grid.inp? (see [here](https://github.com/NOAA-EMC/WW3/blob/25bc4d7a21752587dd6f964ab49a0a2722acbcba/model/inp/ww3_grid.inp#L843-L858))
